tests/networkpolicy_test.go

```diff
@@ -176,11 +267,17 @@ func skipNetworkPolicyRunningOnKindInfra() {
 	}
 }
 
-func createVMICirros(virtClient kubecli.KubevirtClient, namespace string, labels map[string]string) *v1.VirtualMachineInstance {
+//TODO: Change this when [1] is merged using libvmi.NewCirros
```

Comment:
i wouldnt leave this and the following comment here
i would suggest to open a ticket or so

Reply:
Do you mean that we open a issue to refator with `NewCirros`, right ?

Reply:
thats a possibility, whatever you want, either open an issue or not, but i wont leave comments in the code imo please

Reply:
Done